frontend/src/lib/components/modals/first-time-setup/FirstTimeSetupModal.svelte

```diff
@@ -0,0 +1,115 @@
+<script lang="ts">
+  import { mdiTimerSandFull, mdiWeb } from '@mdi/js';
+  import { SlideToggle } from '@skeletonlabs/skeleton';
+  import { T } from '@tolgee/svelte';
+
+  import LanguageSelector from './LanguageSelector.svelte';
+
+  import SvgIcon from '$lib/components/SVGIcon.svelte';
+  import { queueAutoStart } from '$lib/store/settingsStore';
+  import { SetNewUserSetupComplete } from '$wailsjs/go/settings/settings';
+  import { BrowserOpenURL } from '$wailsjs/runtime/runtime';
+
+  export let parent: { onClose: () => void };
+
+  const OpenWelcomeGuide = () => {
+    BrowserOpenURL('https://docs.ficsit.app/satisfactory-modding/latest/ForUsers/Welcome.html');
+  };
+
+  function onClose() {
+    SetNewUserSetupComplete(true);
+    parent.onClose();
+  }
```

Comment:
Pressing escape doesn't trigger this, so the modal should be persistent

Reply:
Pretty sure it is? https://github.com/satisfactorymodding/SatisfactoryModManager/blob/3687f59bd04bcdfbf0bdb37197723e71441bacf9/frontend/src/App.svelte#L200

Reply:
Hmm, I got it to close with escape then, but can't reproduce it anymore